### PR TITLE
Update user guide v1.3

### PR DESCRIPTION
**Pull request by vantemoon**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -230,9 +277,48 @@ TAB data are saved as a JSON file `[JAR file location]/data/addressbook.json`. A
...
-### Archiving data files `[coming in v2.0]`
+### Creating new TAB : `new`
 
-_Details coming soon ..._
+Creates a new TAB and automatically swaps to the newly created data file.
+
+Format: `new`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+This can also be done by clicking `File` - `New Book` or using keyboard shortcut `ctrl + shift + n`
```
Comment (KJunWei) [inline docs/UserGuide.md:287]: `ctrl + shift + n` does not work for me (windows), not sure if its just me...

File: docs/UserGuide.md [hunk 2]
```diff
@@ -230,9 +277,48 @@ TAB data are saved as a JSON file `[JAR file location]/data/addressbook.json`. A
...
+
+Renames the current data file.
+
+Format: `rename {NEW_NAME}`
+
+Renames the current data file as the `NEW_NAME`
+* `NEW_NAME`only accepts alphanumeric, '-' and '_' characters.
+
+Example:
+* `rename {CS2103T}` renames the current data file as `CS2103T.json`
```
Comment (KJunWei) [inline docs/UserGuide.md:310]: `rename {CS2103T}` could instead of `rename CS2103T`

**Review by KJunWei — COMMENTED** [review]
Some minor things that can be edited easily but looks really good otherwise!!

**Review by cxyterence — APPROVED** [review]
LGTM

**Pull request merged**